features/facebookImport/test/importers/posts-importer.test.js

```diff
@@ -37,15 +41,11 @@ describe("Import searches from export with file error", () => {
     });
 
     it("has one error status", async () => {
-        expect(result.length).toBe(1);
+        expect(result.status.length).toBe(1);
```

Comment:
So if it's success, it simply does not go into `status`. Should it be called errorStatus then?

Reply:
In case of success, for the posts importer, the status will be a success status. Otherwise, it will be a list of error or warning statuses.

Reply:
So if it's success result.status.length will also be one?

Reply:
No. At the moment if it is successful status will be a single status object. If there are errors, status will be an array of status objects. When we make statuses explicit objects, we can have a composite status object.

But maybe even now, we could extend the serialisation to either return a single status object of a composite status object with a list of individual statuses

Reply:
The problem is that checking for an implementation-dependent status type, as opposed to simply status object attribute, is a tricky thing, which assumes the consumer is on this convention; they may not.

Reply:
What do you mean in this case by "implementation-dependent status type"? 

Currently for an importer we either get a single status (success, warning, error) or a composes status (a list of statuses for now). Only importers for posts and messages rely on the composed status at the moment to get more details if something fails. But as we will ad more validation to data that might change.

Reply:
Precisely that. It's either a list or a single status, and you need to check in advance which is which, and know what the diferent options are. An object status that would wrap around that and show an uniform interface might be an alternative. Anyway, it's outside the scope of this and the conversation is solved. Thanks!